timeSeriesUtilities/info.json

```diff
@@ -0,0 +1,221 @@
+            "parameters": [
+                {
+                    "title": "Relative Date",
+                    "type": "object",
+                    "name": "relativeDate",
+                    "required": true,
+                    "visible": true,
+                    "editable": true,
+                    "value": null,
+                    "tooltip": "JSON in the format {\"differenceType\": ..., \"differenceValue\": ...}"
+                },
+                {
+                    "title": "Period",
+                    "type": "select",
+                    "name": "period",
+                    "required": true,
+                    "visible": true,
+                    "editable": true,
+                    "value": null,
+                    "options": [
+                        "Hourly",
+                        "Daily",
+                        "Weekly",
+                        "Monthly",
+                        "Quarterly",
+                        "Yearly"
+                    ]
+                },
+                {
+                    "title": "Date Format String",
+                    "type": "text",
+                    "name": "dateFormatString",
+                    "required": true,
+                    "visible": true,
+                    "editable": true,
+                    "value": "YYYY-MM-DDTHH:mm:ss.SSS[Z]",
+                    "tooltip": "How dates are expected to be formatted in the chart JSON"
+                },
+                {
+                    "title": "Existing Times",
+                    "type": "object",
+                    "name": "existingTimes",
+                    "required": false,
+                    "visible": true,
+                    "editable": true,
+                    "value": null,
+                    "tooltip": "The list of times currently shown in the chart, if the chart has previously been generated"
+                },
+                {
+                    "title": "Query Modified",
+                    "type": "checkbox",
+                    "name": "queryModified",
+                    "required": false,
+                    "visible": true,
+                    "editable": true,
+                    "value": false,
+                    "tooltip": "Whether the chart definition has been modified since the last time data was fetched. If so, all time windows will be re-queried and existing data overwritten."
+                }
+            ],
+            "open": false,
+            "category": "utilities"
+        },
+        {
+            "title": "Flatten Data Sets and Groups",
+            "operation": "flatten_data_sets_and_groups",
+            "description": "Takes Data Set and Data Set Group information and outputs the flattened list of data sets to query, a list of Data Sets which should be grouped together, and a dictionary of Data Sets with specified plot types.",
```

Comment:
is this action dependent on any other actions output? if yes, we should mention that too here